lab-geoff/test/gallery-controller-test.js

```diff
@@ -0,0 +1,22 @@
+require('../app/entry.js');
```

Comment:
I think your test problem may also have to do with importing the entry.js file for each test. You want to move those to a test-setup file so they're done once.